src/styles/blocks/card.scss

```diff
@@ -0,0 +1,74 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  color: #060b35;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    align-self: center;
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
```

Comment:
Could you please add the text color to the title and other card texts based on the mockup (it's not pure black) 
![image](https://github.com/mate-academy/layout_product-cards/assets/88306382/78f7028c-2215-4224-98a0-721639ba5b38)